source/_posts/2017/2017-01-10-jak-psat-zajimave-inzeraty-pro-ajtaky.md

```diff
@@ -0,0 +1,195 @@
+- méně textu, více hodnot
+    - co jsou vaše firemní hodnoty? vemte top 5 a dejte je do inzerátu jako klíčová slova do člnku
+
+
+## Co zlepšit na [Jobs.cz](http://lmc.jobs.cz/pridej-se-k-nam/detail/?id=G2-1151391350-aden_brand0&rps=186) 
+
+### Líbí se mi
+
+- *A co tě u nás bude dělat šťastným?* - Parádní nadpis!
+
+- „Místo CV nám klidně můžeš poslat jen link na tvůj profil na LinkedIn“ - super, použití běžného nástroje pro CV! Překvapuje mě, jak málo vidím.
+
+- Líbí se mi, že děláte open-source zaměřený na testování - někam bych tam [Stewarda](https://github.com/lmc-eu/steward) hodil.
+
+- Taky se mi líbí, jak máte v práci velkou monitorovací obrazovku. Tu bych do inzerátu přidal, určitě zaujme a ujistí, že to s testy myslíte vážně. Jinde říkají, že testují, ale když chcete vidět testy, tak nějak nejsou. 
+
+
+### Změnil bych
+
+- Když čtu „znalost PHP 5“, říkám si: myslí 5.3 nebo 5.6? Mají tam projekty na všech těchto verzích? Buď bych napsal konrétní verzi nebo jen PHP.
+
+- Mezi „JS frameworky“ patří Angular 1 a 2, ReactJs, jQuery a další. Umím třeba Angular 1, ale nevím, jestli to stačí. Může se stát, že potřebujete právě někoho přes Angular 1, ale protože to není napsáno, tak se takový člověk nepřihlásí. Radši bych vypsal o které konkrétní  se jedná.
+ 
+- Cokoliv po „spíš“ působí nejistě a nejasně. Upravil bych to do jednoznačného sdělení. 
+
+
+## Co zlepšit na  [Skrz.cz](https://www.startupjobs.cz/nabidka/5101/senior-php-developer-ka-chces-proniknout-do-automatizace-v-projektu-postavenem-na-symfony)
+
+### Líbí se mi
+
+- Líbí se mi tučně zvýrazněné důležité texty, nadpisy působí přehledně a odkazy lákají ke klinutí.
+
+- Pěkně a konkrétně rozepsané, co můžu získat:
+
+    - „společné páteční snídaně“ &dash; mňam - k tomu fotku a už mě máte :).
+    - „ovoce, zelenina, káva a čaj volně dostupné :)“ 
+    „jistotu, domácí prostředí, otevřený přístup, **otevřenou komunikaci**, **prostor k seberealizaci** i **vzdělání**, na kterém si hodně zakládáme“ - tučné body zní fakt dobře, klidně bych je rozepsal. Taky bych nezapomněl na vlastního hapiness managera. 
+
+### Změnil bych
+
+- Spojení „tvrdě makat“ má dost negativní, těžký náboj a může zabarvit zbytek inzerátu. Sám si pod tím představuju přesčasy a nabourávání osobního života. Dokážu tvrdě makat, když v tom vidím smysl. **Pokud tam nějaký smysl je, přidal bych ho hned za tyhle slova.** 
+
+- Nebo bych to, co po programátorovi doopravdy chci. Jde-li třeba o *důraz na kvalitu*: „Píšeš radši kvalitní kód s testy než bastl s technickým dluhem?“
+
+- První 2/3 jsou o firmě, až pak je „Co za to“. Vyměnil bych pořadí a info o firmě zkrátil na  polovic. „Co za to“ zní dost obchodně a rázně. Použil bych spíš „Co pro tebe máme“. 
+
+- Zmešnil bych počet odkazů na polovic. Je jich tam docela dost a jako programátor nevím, jestli si je mám načít, jsou důležité pro práci nebo je to jen reference na nějaký produkt. 
+
+
+## Borci na konci
+
+Na závěr na odlehčení 2 inzeráty, které se mi líbí.
```

Comment:
V copy škole nám říkali, že čísla číslovkami, když to není vyloženě v mluvené řeči nebo tomu blízké.
Líp se to zachytí při skenování.